source/Tipster.cpp

```diff
@@ -280,9 +285,13 @@ Tipster::_LoadSettings()
 void
 Tipster::_ResetTimer()
 {
-	BMessage message(UPDATE_TIP);
-	delete fRunner;
-	fRunner = new BMessageRunner(this, message, fDelay);
+	if (fDelay > 0) {
+	// if delay is set to Off (0), don't send a runner message
+
```

Comment:
This empty line isn't needed.
More importantly: if you go from some delay to "Off", the previous BMessageRunner is still running and keeps changing tips after the the former delay. So you still have to delete the BMessageRunner if delay == 0. Maybe something like this:
```
	if (fRunner != NULL) {
		delete fRunner;
		fRunner = NULL;
	}
	if (fDelay > 0) {
	// if delay is set to Off (0), don't send a runner message
		BMessage message(UPDATE_TIP);
		fRunner = new BMessageRunner(this, message, fDelay);
	}
```

Reply:
Thanks, that's a big help for me! Didn't know how to solve it ...